### DISCUSSION TITLE
Re-enable visibility of documentation tab.

### DISCUSSION THREAD

**Discussion by ENBYSS** [hoppscotch/hoppscotch#2900] [General]
I found out about this project and was very interested in using it. I look at the web portal, and there's no documentation tab - so I assume that I need to self host it to see the tab. I do that, and the tab still isn't there. I spend half an hour trying to figure out what's going on and scouring for information, only to see a discussion post mention that the visibility was disabled due to a refactor.

Now I was interested specifically because of the documentation feature - and am really confused as to why it was disabled. Sure a refactor is incoming, but it's a pretty good feature to have and removing it for almost 2 months now kind of leaves a sour taste in my mouth.

Especially because there's no note of this anywhere besides the github discussions board - I couldn't find anywhere in the documentation or readme about this, and at least it would've been nice to have a small message saying it's temporarily disabled.

Could it be re-enabled? Or at the very least, could an update to the README/Documentation happen to note that it's currently unavailable?

**Comment by pycharmer1221** [comment]
i am also wondering the same thing so far hoppscotch is the best solution that i can find that suits my requirement.
but only the documentation feature is missing.

please reply here if you found any solution of enabling it.
